docs/using-the-sdk/cosmovisor.md

```diff
@@ -0,0 +1,174 @@
+# Cosmosvisor Quick Start
+
+`cosmovisor` is a small process manager around Cosmos SDK binaries that use the upgrade module to allow
```

Comment:
nit: `cosmovisor` is a small process manager around Cosmos SDK binaries that **uses** the upgrade module to allow

Reply:
fixed, please check again

Reply:
ace :bow: